editor/src/clj/editor/pipeline/texture_set_gen.clj

```diff
@@ -153,7 +153,10 @@
 
 (defn atlas->texture-set-data
   [animations images margin inner-padding extrude-borders max-page-size]
-  (let [sprite-geometries (mapv make-image-sprite-geometry images)]
+  ;; NOTE: Images order matters when generating the layouts, especially if they are of the same size.
+  ;; Since the SHA1 for the packed-page-images-generator is insensitive to order, things could get out of sync
+  (let [images (sort-by #(-> % :path :project-path) images)
```

Comment:
The `:path` seems to be a `Resource`? In that case, we should use `resource/proj-path` (throws for `nil`) or `resource/resource->proj-path` (returns "" for `nil`) instead of accessing the `:project-path` field of the resource directly. A `ZipResource` does not even have this field, so we must use the methods on the `Resource` protocol whenever we want information from a `Resource`.